### ISSUE TITLE
author page miguel alvarez not on list

### ISSUE THREAD

**Issue by stefaniebutland** [ropensci/roweb3#91]
At https://ropensci.org/author/, this author https://ropensci.org/author/miguel-alvarez/ is not listed.

I can't think of why.

**Comment by maelle** [comment]
Because he only authored a talk. Same for Rebecca Killick. I'll find a workaround.

**Comment by maelle** [comment]
Actually it'd demand some more design decisions.

* Do we want to have speakers in the author list?
* Only when there is an associated talk (resources) or would the event be sufficient?
* On the individual author page how do we list events?
* On the individual author page how do we list talk resources?

**Comment by maelle** [comment]
I think it's sort of a duplicate of #16?

**Comment by stefaniebutland** [comment]
Thanks for the explanation. I don't think anything further should be done until the team decides we are ready to add more important things like packages, reviews etc to author pages. 

What's the best way to add your design decisions https://github.com/ropensci/roweb3/issues/91#issuecomment-730987333 to #16 and then close this issue?
